composer.json

```diff
@@ -112,6 +113,9 @@
             },
             "drupal/views_infinite_scroll" : {
                 "Headers in table format repeat on load more instead of adding rows": "https://www.drupal.org/files/issues/2019-08-15/table_tbody_append-2899705-26.patch"
+            },
+            "webonyx/graphql-php": {
```

Comment:
This does require ext-mbstring, which Drupal doesn't. Drupal uses symfony polyfill for this. Not sure if that causes any harm 🤔 could potentialy cause some issues for existing installions I guess 🤔

Reply:
https://www.drupal.org/docs/system-requirements/php-requirements#s-mbstring-

Drupal says it does require it, also if it's not installed it seems to break https://www.drupal.org/node/2560123
So it should already be used everywhere.